### ISSUE TITLE
Project needs a logo

### ISSUE THREAD

**Issue by quaid** [theopensourceway/the-project#13]
I'd love to get some stickers for e.g. SCaLE on 05 March, but can we think of a logo by then?

@tigert has been invited to this group, let's see what he things.

What the brand of this project conveys, IMO:
* Expert practitioners teaching each other and everyone how to practice the open source way
* Concise, actionable content -- you learn not just what but how.
* Thoughtful reasoning -- you learn not just what and how but why.
  * It's the smashing of cargo cult thinking.
* Specific to open source software yet generic enough concepts that they can apply elsewhere wishing to practice the open source way beyond software.
* Upstream for any number of efforts:
  * For opensource.com definition of "the open source way"
  * Knowledge base at community.redhat.com
* ...

**Comment by tigert** [comment]
Yeah. I have poked at a few ideas, and I am inviting some more thought to the process. 

My initial thoughts were the kind of obvious "road to horizon" idea:
![image](https://user-images.githubusercontent.com/2587818/90383863-681b9180-e089-11ea-8436-d897bf9a8e10.png)

And maybe a bit more abstract one:
![image](https://user-images.githubusercontent.com/2587818/90383910-75d11700-e089-11ea-8eb6-0541a37cd1d5.png)

But I am not yet 100% happy with this idea. Does anyone get any good thoughts from these sketches?

**Comment by semioticrobotic** [comment]
I do find this direction appealing and would not be at all upset if we pursued it.

I might also suggest something that emphasizes the "craft" and practice-focused spirit of the project—something, perhaps, that communicates the notion that building open source communities is itself a practice that must be continually built and shared, a practice we're all committed to developing together. I'm not sure what that might be, however (and most of the ideas that spring to my mind, like iconography featuring analog tools or construction apparatus seem trite to me).

**Comment by quaid** [comment]
Definitely like the look there, in and of itself it's a cool treatment. Definitely worth thinking about the 'way' aspect as a word with many applicable meanings. Maybe in the sense of the Appian Way and other historical roads.

Another idea that just came to me is to riff off how as a community of practice we are doing knowledge curation of something that is constantly evolving. What them came to mind is a library book cart, such as is used to return books to the shelves, as a metaphor. Or even a mobile library.

Library book cart drill teams:

https://www.allpurposeguru.com/2012/12/presenting-library-book-cart-drill-teams/

Bookmobiles are great and quite an old concept:

https://en.wikipedia.org/wiki/Bookmobile

Which leads to me imagining a cute bookmobile icon rolling down a highway. :)

Any travelling knowledge sharer would be good inspiration here.

Side thought, the dashed-line strip says 'road' to me, is that relatively universal? Seems so, but another one that occurs to me is the road or path over rolling hills:

https://www.google.com/search?q=road+over+rolling+hills

**Comment by tigert** [comment]
@semioticrobotic and @quaid  yeah, I feel the same way, it "works" but it could accommodate a bit of "maker" spirit that happen along the road somehow. 

Good question about the road metaphor too.

Anyway, just wanted to let you know I'm this far in the process. Thanks for the feedback, I'll sketch something and we can continue!

**Comment by tigert** [comment]
Hm, one way to do this would be something like this. Obviously we'd need a bit better illustrations of the people, but you get the idea from this.

![image](https://user-images.githubusercontent.com/2587818/100888165-7b2e6200-34be-11eb-9b16-9cd18a682478.png)

**Comment by tigert** [comment]
Or even this. On a book cover design the "road" could go around the spine into the back even.

![image](https://user-images.githubusercontent.com/2587818/100888632-027bd580-34bf-11eb-86ff-7154581bca96.png)

**Comment by quaid** [comment]
I wonder if it's too imperialist to go into the Western classics and do some brainstorming around the [Appian Way](https://www.google.com/search?q=appian+way&client=firefox-b-1-d&source=lnms&tbm=isch&sa=X&ved=2ahUKEwiAj5Gfw7LtAhXILs0KHQnXBpEQ_AUoAXoECA8QAw&biw=1920&bih=948), which was ancient Rome's greatest road (of the ones that all-lead-to-Rome). Or possibly the [Silk Road](https://www.google.com/search?q=the+silk+road&client=firefox-b-1-d&source=lnms&tbm=isch&sa=X&ved=2ahUKEwipg4atxLLtAhVMCM0KHYR0Du8Q_AUoAXoECBcQAw&biw=1920&bih=948) is another visual/thematic touchstone.

There is a point in here where we have the clash of multiple meanings. There is the road meaning, there is also the meaning exemplified in the Japanese suffix "-do", which can be translated aiui to "the way of ...". For example, Aikido is "the way (do) of harmonizing (ai) energy (ki)".

Arguably, the latter meaning is more what is meant by "the open source way", but in English, there is a shared meaning in that "the road taken" is a metaphor that is more inline with "the way of" interpretation.

**Comment by quaid** [comment]
I propose that we go ahead with a project branding that is in the best shape we can have it by 15 Dec. I know it's not normal to iterate often on a community brand, but we need something to start with, and I think next year we'll have more community involvement toward something that is meaningful for the people who are here. It will therefore be OK to have something that is good enough at conveying these messages.

@tigert what is a good deadline for you for "stop giving me ideas, I'll finish with what we have right now?" It's OK if the answer is, "Right now, thanks," because we may have a few artifacts we want and don't want to run out of time to test them in the book build and on the website.

I think we mainly need the logo and a favicon? I see you are working in monochrome here, and I wonder if that is a good way to stay for now -- don't pick colors outside of the colors-of-typeface-in-a-book?

For the [website](https://www.theopensourceway.org/), I don't see any need to redesign what is there -- we would drop the logo in the top left-corner for all pages, including the [HTML of the built book](https://www.theopensourceway.org/preview-1.99-book.html). (I'll look into how to do that in Asciidoctor.)

Same for the PDF, I'm hoping between myself and @shaunix we can sort out how to brand the [frontpage of the PDF](https://www.theopensourceway.org/preview-1.99-book.pdf).

**Comment by quaid** [comment]
In particular, we could use a beta logo for the top of these pages:

https://www.theopensourceway.org/
https://www.theopensourceway.org/the_open_source_way-guidebook-2.0.html

The latter can be hacked in, or we can have a template for page that Asciidoctor builds the <body> elements to drop into (the template had <header> and <meta> and any footer content, aiui).

Is there an idea you think is interesting enough to try as a beta?

**Comment by quaid** [comment]
Identified needs:

* Color palette
* Font for title, headers, text
* Consistent style across HTML, PDF, and website
* EPub perhaps
* Full book cover illustration
* ...

**Comment by tigert** [comment]
Here is one idea with the road existing in the negative space between the lines of text.

![image](https://user-images.githubusercontent.com/2587818/107492180-ccc92c00-6b94-11eb-8864-038b84b5064b.png)

**Comment by tigert** [comment]
Something very different, an idea around git workflow visualization and some art decoish elements maybe? :-)

![image](https://user-images.githubusercontent.com/2587818/107518416-a2d53100-6bb7-11eb-9611-4e89958b9527.png)

Softer, more organic which I like.

**Comment by semioticrobotic** [comment]
Dang, @tigert! You make choosing very difficult.

I think the subtly of the second design is what draws me to it. I'd vote for that one.

**Comment by quaid** [comment]
@tigert OK, let's go with the logo in https://github.com/theopensourceway/theproject/issues/13#issuecomment-776718288 (Metro git merge style), and then make quick decisions on the fonts and colors that go with that.

Can you and @shaunix choose (with me and whoever else joins in) on #67 by end of Friday 19 February?

**Comment by semioticrobotic** [comment]
Closing as complete, since we adopted @tigert's [logo](https://github.com/theopensourceway/the-project/issues/13#issuecomment-776718288) design.
